.pre-commit-config.yaml

```diff
@@ -8,7 +8,7 @@ ci:
 
 repos:
   - repo: https://github.com/pre-commit/pre-commit-hooks
-    rev: v4.3.0
+    rev: v4.2.0
```

Comment:
why downgrading in this file ? better rebase with master and not touch this file

Reply:
OK